x-pack/plugins/fleet/server/index.ts

```diff
@@ -45,7 +45,6 @@ export const config: PluginConfigDescriptor = {
     registryProxyUrl: schema.maybe(schema.uri({ scheme: ['http', 'https'] })),
     agents: schema.object({
       enabled: schema.boolean({ defaultValue: true }),
-      fleetServerEnabled: schema.boolean({ defaultValue: false }),
```

Comment:
we probably want to add this as `deprecated` to avoid breaking change for environment that will use that flag it could be done here https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/server/index.ts/#L40-L43

Reply:
Ahhh. yes - forgot about that. I'll go back to your last set of changes (where you had removed this flag) and make sure I sync this up. Thanks for the feedback.